envoy-control-runner/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/debug/SnapshotDebugController.kt

```diff
@@ -47,6 +48,20 @@ class SnapshotDebugController(controlPlane: ControlPlane) {
      * It contains the versions of XDS resources and the contents for a provided node JSON
      * extracted from Envoy's config_dump endpoint.
      */
+    @PostMapping("/snapshotV2")
+    fun snapshot(@RequestBody node: io.envoyproxy.envoy.api.v2.core.Node): ResponseEntity<SnapshotDebugInfo> {
```

Comment:
`io.envoyproxy.envoy.api.v2.core.Node` could be aliased

Reply:
Done